src/components/RegisterForm.js

```diff
@@ -1,18 +1,38 @@
 import React, { useState } from 'react'
 import { Link } from 'react-router-dom'
+import { registerUser } from "../utils/requests";
 
-export default function RegisterForm({ login, error }) {
+export default function RegisterForm() {
+	const [info, setInfo] = useState('')
 	const [details, setDetails] = useState({
-		username: '',
+		userName: '',
 		email: '',
 		password: '',
 		phoneNumber: '',
 		role: '',
 	})
 
-	const submitHandler = (e) => {
+	const onSubmit = async (e) => {
 		e.preventDefault()
-		login(details)
+
+		try {
+			const isRegistered = await registerUser(details)
+			if (isRegistered) {
+				setInfo(isRegistered)
+				setDetails({
+					userName: '',
+					email: '',
+					password: '',
+					phoneNumber: '',
+					role: ''
+				})
+				setTimeout(() => { setInfo('') }, 3000)
```

Comment:
I don't really understand the purpose of this line of code

Reply:
I have a <div> as information for user. When I have something in 'info' a message appears to the user, and after 3 seconds 'info' became empty and message is hidden